staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook.go

```diff
@@ -109,11 +109,24 @@ func (w *WebhookTokenAuthenticator) AuthenticateToken(token string) (user.Info,
 	}, true, nil
 }
 
+// NOTE: client-go doesn't provide a registry. client-go does registers the
+// authentication/v1beta1. We construct a registry that acknowledges
+// authentication/v1beta1 as an enabled version to pass the test in
+// NetGenericWebhook.
```

Comment:
I am confused about the mention of the test when this appears to be referenced in real code?